templates/ubiquiti_edgeswitch_show_version.textfsm

```diff
@@ -0,0 +1,20 @@
+Value VERSION (.*)
+Value SWITCH_TYPE (\S+\s\S+\s\S+)
+Value SWITCH_MODEL (\S+)
+Value SERIAL (\S+)
+Value MAC (\S+)
+Value HW_REVISION (\S+)
+
+Start
+  ^Switch:.+
+  ^Machine\s+Type\.+\s+${SWITCH_TYPE}
+  ^Machine\s+Model\.+\s+${SWITCH_MODEL}
+  ^Serial\s+Number\.+\s+${SERIAL}
+  ^Burned\s+In\s+MAC\s+Address\.+\s+${MAC}
+  ^Software\s+Version\.+\s+${VERSION}
+  ^Hardware\s+Revision\.+\s+${HW_REVISION}
+  # netmiko's send_command starts command output with the last symbol from command sent
+  ^\w
```

Comment:
Could you also do the following:

```suggestion
  ^\w$$
```

This will help limit what it matches on and matches single character we're looking for.

Reply:
It matches unneccesary lines, such as "System Description.."
^\w$$ will lead to error rising.

Reply:
Ah yes, that is what we're looking for. To avoid that, we want to be explicit in what we're just accounting for, but not capturing.

You'd just add a new line such as: `  ^System\s+Description` and the beginning of any other lines that may need to be accounted for, but not necessarily captured.

Reply:
I got it, thanks!